controllers/notes.js

```diff
@@ -1,9 +1,38 @@
-exports.getNoteById = (req, res, next) => {
-        res.status(200).json({
-        success: true,
-         msg: `Note with id: ${req.params.id}`
-    });
+/*These controllers, when the endpoints are ChannelSplitterNode, return JSON, or wahtever */
+
+//this file contains the functions of the app
+
+exports.updateNote = (req, res, next) => {
+  Note.findByIdandUpdate(req.params.ide, req.body, {runValidators:true }, ()=>{
```

Comment:
```suggestion
Note.findByIdAndUpdate()```

And probably is a type -- 
```suggestion
req.params.id
```